vpr/src/draw/search_bar.cpp

```diff
@@ -123,8 +144,15 @@ void search_and_highlight(GtkWidget* /*widget*/, ezgl::application* app) {
     else if (search_type == "Net Name") {
         std::string net_name = "";
         ss >> net_name;
+        AtomNetId atom_net_id = AtomNetId::INVALID();
+        atom_net_id = atom_ctx.nlist.find_net(net_name);
```

Comment:
Can collapse these two lines I think (no need for invalid preset).
Should also comment why you only need to look through the atom net names (all clb nets are a subset of atom net names)?